src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -85,39 +90,27 @@ bool EnableCgroupSubtreeControl(const char *subtree_control_path) {
 
 namespace internal {
 
-bool CanCurrenUserWriteCgroupV2() { return access("/sys/fs/cgroup", W_OK | X_OK) == 0; }
+bool CanCurrenUserWriteCgroupV2() {
+  return access(kCgroupDirectory.data(), W_OK | X_OK) == 0;
+}
 
-bool IsCgroupV2MountedAsRw() {
-  // Checking all mountpoints directly and parse textually is the easiest way, compared
-  // with mounted filesystem attributes.
-  std::ifstream mounts("/proc/mounts");
-  if (!mounts.is_open()) {
+bool IsCgroupV2MountedAsRw(const std::string &path) {
+  // Check whether cgroupv2 is mounted solely at `/sys/fs/cgroup`.
```

Comment:
This comment appears to be out of date since the function takes a `path`

Reply:
Updated